### PR TITLE
Add a welcome message for openshift/api PRs

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@deads2k @JoelSpeed ptal.  Any other information you'd like to convey in this message?  it will be posted to all api PRs.

see: https://github.com/openshift-assisted/assisted-installer-deployment/pull/219#issuecomment-1149725287 for an example of how it functions in another repo.

File: core-services/prow/02_config/openshift/api/_pluginconfig.yaml [hunk 1]
```diff
@@ -18,3 +18,21 @@ plugins:
   openshift/api:
     plugins:
     - approve
+welcome:
```
Comment (osherdp) [inline core-services/prow/02_config/openshift/api/_pluginconfig.yaml:21]: Don't forget to also add the plugin (either here or globally in openshift org):
```suggestion
    - approve
    - welcome
welcome:
```

**Review by osherdp — CHANGES_REQUESTED** [review]

File: core-services/prow/02_config/openshift/api/_pluginconfig.yaml (hunk 1 above)
Comment (bparees) [reply core-services/prow/02_config/openshift/api/_pluginconfig.yaml]: thanks, fixed

File: core-services/prow/02_config/openshift/api/_pluginconfig.yaml [hunk 2]
```diff
@@ -18,3 +18,22 @@ plugins:
...
+
+    ``bugzilla/valid-bug`` - applied if your PR references a valid bugzilla bug
+
+    OR
+
+    ``qe-approved``, ``docs-approved``, and ``px-approved`` - these labels can be applied by anyone in the openshift org via the ``/label`` command.
+
+    Who should apply these qe/docs/px labels?
+    - For a no-Feature-Freeze team who is merging a feature before code freeze, they need to get those labels applied to their api repo PR by the appropriate teams (i.e. qe, docs, px)
+    - For a Feature Freeze(traditional) team who is merging a feature before FF, they can self-apply the labels(via /label commands), they are basically irrelevant for those teams
```
Comment (JoelSpeed) [inline core-services/prow/02_config/openshift/api/_pluginconfig.yaml:36]: Nit, should have a couple of extra spaces
```suggestion
    - For a Feature Freeze (traditional) team who is merging a feature before FF, they can self-apply the labels (via /label commands), they are basically irrelevant for those teams
```

**Review by JoelSpeed — COMMENTED** [review]
I think, for this particular repository, it would also be helpful to link out to the openshift API conventions, perhaps we can add something along the lines of

> API design plays an important part in the user experience of OpenShift. If you haven't already done so, please review the [OpenShift API Conventions](https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md) and ensure that your proposed changes are compliant. 

This block is missing the value to the author though, I think it needs to say something like `Doing so will make the API review process faster` or `Doing so helps API reviewers to provide more constructive and useful feedback` but I'm not sure I like the wording

File: core-services/prow/02_config/openshift/api/_pluginconfig.yaml (hunk 2 above)
Comment (bparees) [reply core-services/prow/02_config/openshift/api/_pluginconfig.yaml]: fixed

**Comment by bparees** [comment]
> I think, for this particular repository, it would also be helpful to link out to the openshift API conventions, perhaps we can add something along the lines of

added:
```
API design plays an important part in the user experience of OpenShift and as such API PRs are subject to a high level of scrutiny to ensure they follow our best practices. If you haven't already done so, please review the [OpenShift API Conventions](https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md) and ensure that your proposed changes are compliant.  Following these conventions will help expedite the api review process for your PR.
```

**Comment by JoelSpeed** [comment]
> > I think, for this particular repository, it would also be helpful to link out to the openshift API conventions, perhaps we can add something along the lines of
> 
> added:
> 
> ```
> API design plays an important part in the user experience of OpenShift and as such API PRs are subject 
> to a high level of scrutiny to ensure they follow our best practices. If you haven't already done so, please
> review the [OpenShift API Conventions](https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md) and ensure that your proposed changes are compliant.  Following these conventions will help
> expedite the api review process for your PR.
> ```

Thanks, that's nicely worded!

**Pull request merged**